src/Hearthrock.Contracts/RockMinion.cs

```diff
@@ -1,132 +1,138 @@
-        /// </summary>
-        public bool IsFrozen { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion is exhausted.
-        /// </summary>
-        public bool IsExhausted { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion is asleep.
-        /// </summary>
-        public bool IsAsleep { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion is stealth-ed.
-        /// </summary>
-        public bool IsStealthed { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion can attack.
-        /// </summary>
-        public bool CanAttack { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion can be attacked.
-        /// </summary>
-        public bool CanBeAttacked { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion has taunt.
-        /// </summary>
-        public bool HasTaunt { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion has wind-fury.
-        /// </summary>
-        public bool HasWindfury { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion has divine shield.
-        /// </summary>
-        public bool HasDivineShield { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion has aura.
-        /// </summary>
-        public bool HasAura { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion has enraged.
-        /// </summary>
-        public bool IsEnraged { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion has trigger visual.
-        /// </summary>
-        public bool HasTriggerVisual { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion has inspire.
-        /// </summary>
-        public bool HasInspire { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion has deathrattle.
-        /// </summary>
-        public bool HasDeathrattle { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion has battlecry.
-        /// </summary>
-        public bool HasBattlecry { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion has lifesteal.
-        /// </summary>
-        public bool HasLifesteal { get; set; }
-
-        /// <summary>
-        ///  Gets or sets a value indicating whether the minion is poisonous.
-        /// </summary>
-        public bool IsPoisonous { get; set; }
-    }
-}
+﻿// <copyright file="RockMinion.cs" company="https://github.com/yangyuan">

```

Comment:
It's very weird that git says entire this file is changed.
Probably caused by complex commits relations.

Do you mind to rebase and squash the changes into 1 commit? (Which is a required practice in many big companies)

Reply:
oh, yes, sure, I will do it asap